### PR TITLE
Fixes #1540 - Appends the proper payload to the creationButtonClicked event

### LINKED ISSUE #1540: ManagementToolbar event creationButtonClicked does not contain the proper payload
In ManagementToolbar, the `creationButtonClicked` event is not dispatched with the proper payload. For consistency, it should contain the configured item.

### PR DESCRIPTION
**Pull request by jbalsas**

### CODE REVIEW

**Comment by matuzalemsteles** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: packages/clay-management-toolbar/src/ClayManagementToolbar.js [hunk 1]
```diff
@@ -77,14 +77,32 @@ class ClayManagementToolbar extends ClayComponent {
...
+	 * Continues the propagation of the plus button clicked event. This path only
+	 * happens when there's a single primary item. Thus, the data payload associated
+	 * with the event is simply the first primary item.
 	 * @param {!Event} event
 	 * @private
 	 * @return {Boolean} If the event has been prevented or not.
 	 */
 	_handleCreationButtonClicked(event) {
+		const creationMenu = this.creationMenu;
+		const primaryItems = creationMenu.primaryItems;
```
Comment (matuzalemsteles) [inline packages/clay-management-toolbar/src/ClayManagementToolbar.js]: hey @jbalsas we have to do a better check here, this event is also triggered when we do not have a `creationMenu`.

https://github.com/liferay/clay/blob/2.x-develop/packages/clay-management-toolbar/src/ClayManagementToolbar.soy#L501

Comment (jbalsas) [reply packages/clay-management-toolbar/src/ClayManagementToolbar.js]: good catch! I assumed we wouldn’t be firing this if we didn’t have a menu... could you please fix it and resend this?

Thanks!

**Comment by matuzalemsteles** [comment]
Resent #1542

**Pull request closed without merging**